atomicapp/templates/nulecule/answers.conf.sample.tpl

```diff
@@ -0,0 +1,7 @@
+[general]
+namespace = default
+provider = kubernetes
+
+[${app_name}]
+image = centos/httpd
+
```

Comment:
Do we need a template answers.conf.sample file? Since this should be generated by atomicapp from the Nulecule I see no reason why we would need one.